src/scripts/painter.json

```
@@ -0,0 +1,27 @@
+{
+  "title": "Painter",
+  "description": "Add colour to your dashboard posts",
+  "icon": {
+    "class_name": "ri-brush-line",
+    "color": "white",
+    "background_color": "#4bb51b"
+  },
+  "preferences": {
+    "own": {
+      "type": "text",
+      "label": "Own post colour"
+    },
+    "original": {
+      "type": "text",
+      "label": "Original post colour"
+    },
+    "reblog": {
+      "type": "text",
+      "label": "Reblogged post colour"
+    },
+    "liked": {
+      "type": "text",
+      "label": "Liked post colour"
+    }
```

just tested this out, and all these preference values default to the string `"undefined"` for... probably obvious reasons

it [is required that preferences have a `"default"` value](https://github.com/AprilSylph/XKit-Rewritten/blob/master/docs/framework.md#preferences-preference-name-default), so it should probably just be an empty string for each preference here

```suggestion
    "own": {
      "type": "text",
      "label": "Own post colour",
      "default": ""
    },
    "original": {
      "type": "text",
      "label": "Original post colour",
      "default": ""
    },
    "reblog": {
      "type": "text",
      "label": "Reblogged post colour",
      "default": ""
    },
    "liked": {
      "type": "text",
      "label": "Liked post colour",
      "default": ""
    }
```